mapie/conformity_scores/conformity_scores.py

```diff
@@ -158,20 +188,154 @@ def get_conformity_scores(
 
         Parameters
         ----------
-        y: NDArray
-            Observed values.
+        X: NDArray of shape (n_samples_calib, n_features)
+            Observed feature values.
+
+        y: NDArray of shape (n_samples_calib,)
+            Observed target values.
 
-        y_pred: NDArray
-            Predicted values.
+        y_pred: NDArray of shape (n_samples_calib,)
+            Predicted target values.
 
         Returns
         -------
-        NDArray
+        NDArray of shape (n_samples_calib, 1)
             Conformity scores.
         """
-        conformity_scores = self.get_signed_conformity_scores(y, y_pred)
+        conformity_scores = self.get_signed_conformity_scores(X, y, y_pred)
         if self.consistency_check:
-            self.check_consistency(y, y_pred, conformity_scores)
+            self.check_consistency(X, y, y_pred, conformity_scores)
         if self.sym:
             conformity_scores = np.abs(conformity_scores)
         return conformity_scores
+
+    @staticmethod
+    def get_quantile(
+        values: NDArray,
```

Comment:
UPD: prefer the term `conformity_scores` as done before in the file.

Reply:
By doing that, it will be aligned with your docstring.

Reply:
I understood that you wanted me to change the name of the parameter in `get_estimation_distribution` because in this method the parameter is either the conformity scores or the quantile of them -> the conformity scores in both situations.
In `get_quantile` the parameter `values` is sometimes the conformity scores and sometimes the conformity scores aggregated with the predictions (according to the class formula), wouldn't it be confusing naming those two values "conformity_scores" ?

Reply:
If I have a choice between `values` and `conformity_scores`, I prefer `conformity_scores` for greater clarity.

Reply:
Confusion can arise if we don't understand the shape of the input. This should be checked in such cases.

Reply:
Done